### DISCUSSION TITLE
I am a student, and I do not have access to copilot

### DISCUSSION THREAD

**Discussion by Laohay** [community/community#62459] [GitHub Education]
### Select Topic Area

Question

### Body

hello, I am a student, I had access to the github student developer pack by declaring my school etc, when I am on vs code I install copilot which I am asked to join to my github account to take advantage of its free access as a student , but on my github account it does not unlock the free one on the contrary it asks me to create an organization to pay for the purchase of copilot monthly when it is normally free for me, please help me I really need copilot

**Comment by IgnacioPalma** [comment]
Hello @Laohay :wave:

I just watched you comment on the other post xD

You can follow the steps provided on this [guide](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279) in order to have access to Copilot. 
You can activate Copilot by going to the page of Copilot (https://copilot.github.com/?WT.mc_id=academic-88217-leestott) and selecting “Start Free Trial”, you will be prompted that you have Copilot for free as a Student, then you can follow the steps.

Let me know if this helps.

**Comment by rushabhcodes** [comment]
1. Sign up for GitHub: If you haven't already, create a GitHub account using your student email address.
2. Verify your student status: Go to the GitHub Education website and follow their instructions to verify your student status using your college email ID.
3. Apply for GitHub Education benefits: Once you've verified your student status, apply for GitHub Education benefits, which may include access to GitHub Copilot.
(It may take 2 working days to get verified)
5. Check your benefits: After your application is approved, you should receive an email or notification confirming your eligibility and listing the benefits you can access. This may include access to GitHub Copilot if it's part of the offer.
6. Access GitHub Copilot: If GitHub Copilot is one of the benefits you are eligible for, you will be able to use it while coding in your GitHub repositories.

**Comment by samus-aran** [comment]
Hi @Laohay (after mixing this post with another).

It looks like you should be able to utilise the Copilot for Students benefits. Let us know if you are still having issues.

**Reply by Laohay** [reply]
hi, @samus-aran i still have the same issue even if the support told that it's fixed , i am waiting

**Reply by royayon** [reply]
@samus-aran  I have got approved for student benefits. but I still can not use copilot.

**Comment by JirkaUlbricht** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> hello, I am a student, I had access to the github student developer pack by declaring my school etc, when I am on vs code I install copilot which I am asked to join to my github account to take advantage of its free access as a student , but on my github account it does not unlock the free one on the contrary it asks me to create an organization to pay for the purchase of copilot monthly when it is normally free for me, please help me I really need copilot

If you have been approved then you have to go to [https://github.com/github-copilot/free_signup](url) and there you have to press Get access Github copilot and set your preffered copilot settings and click the button down below and you are done. :)

![image](https://github.com/user-attachments/assets/602f8fae-efe9-44e6-a4e9-902dacbfe489)

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
